go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -810,6 +916,13 @@ func (vc *VitessCluster) GetVTGateConn(t *testing.T) *mysql.Conn {
 	return getConnection(t, vc.ClusterConfig.hostname, vc.ClusterConfig.vtgateMySQLPort)
 }
 
+func getVTGateConn() (*mysql.Conn, func()) {
```

Comment:
A quick comment would be nice. Just so that it's easy to see w/o looking at the function that it returns a connection (obvious) and a function to close it in a defer.